pkg/test-infra/cdc/operation.go

```diff
@@ -84,6 +89,9 @@ func (c *CDCOps) Delete(tc *Recommendation) error {
 }
 
 func (c *CDCOps) ApplyCDC(cc *CDC) error {
+	if err := c.applyObject(cc.Service); err != nil {
```

Comment:
why applying CDC job before creating CDC object?

Reply:
Why do you mean `CDC job` here? I apply that job after this function

Reply:
my fault.